tests/integration/base_test.py

```diff
@@ -42,24 +56,30 @@ def popup_check(self, mentoring, item_feedbacks, do_submit=True):
 
         submit = mentoring.find_element_by_css_selector('.submit input.input-main')
 
+        scenario_title = mentoring.find_element_by_css_selector('h2.main')
+
+        question_title = mentoring.find_element_by_css_selector('h3.question-title')
+
         for index, expected_feedback in enumerate(item_feedbacks):
             choice_wrapper = choices_list.find_elements_by_css_selector(".choice")[index]
             if do_submit:
                 # click on actual radio button:
                 choice_wrapper.find_element_by_css_selector(".choice-selector input").click()
                 submit.click()
+                self.wait_until_exists('.choice-result.checkmark-incorrect,.choice-result.checkmark-correct')
             item_feedback_icon = choice_wrapper.find_element_by_css_selector(".choice-result")
-            choice_wrapper.click()
+            # choice_wrapper.click()
```

Comment:
@jbzdak if you are happy with how the functionality is tested here then I think it's fine to remove. If you want to make sure you could ask @e-kolpakov as it was introduced in his commit here: https://github.com/open-craft/xblock-mentoring/commit/21f1beab6549afd5979b327c7659be2119b745d0#diff-11e3279e8a9044e3a2707b57f7ec8e9cR175